### DISCUSSION TITLE
Category xml getting corrupted

### DISCUSSION THREAD

**Discussion by mariosm1** [percussion/percussioncms#903] [Troubleshooting]
In some cases, categories that include accented characters or other special characters can cause the category.xml to not save correctly.  This will occur when editing categories in the CMS through the Admin panel.  It is a good idea to back up the category.xml on the server side regularly with a cron task until we have a reproducible case.

On Windows servers, you can configure a task by opening up the Task Scheduler.  Create a new task to copy the Percussion/rx_resources/category/category.xml to a backup location.

On Linux systems, a simple cron like the one below will work, which will back up the file every day at 3pm.  Run `crontab -e` in the terminal, then paste in the following on a new line, making sure to update the paths.
0 15 * * * /bin/cp /path/to/Percussion/rx_resources/category/category.xml  /backup-location/

**Comment by natechadwick** [comment]
@matomario8  Do you have an example of characters that may cause this behavior?  if you do we can convert this to an issue and fix it.
